app/mailers/reminder_mailer.rb

```diff
@@ -0,0 +1,27 @@
+class ReminderMailer < MandrillMailer::TemplateMailer
+  default from: "reminder@verba-app.com"
+
+  def send_reminder(user)
+    mandrill_mail(build_mail(to: [user],
+                             reminder_copy: ReminderCopy.copy_for(user)))
+  end
+
+  private
+  def build_mail(fields)
+    reminder_copy = fields.delete(:reminder_copy)
+
+    # Required fields
+    fields[:to].map!{ |user| {email: user.email, name: user.username} }
+    fields[:template] = 'daily-reminder'
```

Comment:
So, this will look in Mandrill to see if I have created a template called 'daily-reminder', correct?